examples/vulkan/Vulkan.cpp

```diff
@@ -1432,7 +1433,7 @@ class VulkanExample
         VkBuffer       stagingBuffer       = {};
         VkDeviceMemory stagingBufferMemory = {};
 
-        if (!createBuffer(sizeof(vertexData),
+        if (!createBuffer(vertexData.size(),
```

Comment:
This is not correct, this gives the vertex count but not the size of the vertex this should be multiplied by `sizeof(vertexData[0]);` or continue using `sizeof`